docker/chromium/base/Dockerfile

```diff
@@ -32,6 +32,7 @@ RUN dpkg --add-architecture i386 && \
         libdconf1 \
         libdconf1:i386 \
         libgconf-2-4:i386 \
+        libgm-dev:i386 \
```

Comment:
There's a typo here: should be `libgbm-dev:i386` (missing `b` in `gbm`). I'll send a PR.

Reply:
Thanks!